### DISCUSSION TITLE
Transparent to wallpaper

### DISCUSSION THREAD

**Discussion by tipa40** [hyprwm/Hyprland#1979] [Q&A] [answered]
Hi,
Is it possible to, and if so will you ever, implement a transparency that would for every window be towards the background wallpaper directly, not showing/”ignoring” any intermediate, windows/layers? I would think this feature would be really cool because I would like my wallpaper always to be visible, at least from my terminal windows.

Or is it achievable already?

**Comment by Duckonaut** [comment, marked as answer]
It's a thing already, `decorations:blur_xray`. Check out the relevant section in Variables in the wiki

**Reply by tipa40** [reply]
That's awesome, thank you very much! :)

**Comment by tipa40** [comment]
Really nice! 
Unfortunately for me though, my usage scenario is for the special workspace/scratchpad.
I guess there is today no way to accomplish the same for that layer, is there?

**Comment by hoarse-boy** [comment]
what is the new setting for hyprland 0.54 and up? the setting `decorations:blur_xray` return an error as it is not exist anymore. would be better to work for special workspace too like @tipa40 said.

**Comment by hoarse-boy** [comment]
found it. enabling blur and xray. this is for the hyprland v0.54

```config
decoration {
  blur {
    # enabled = false
    enabled = true
    size = 0 
    passes = 0 
    vibrancy = 0.5
    xray = true
    new_optimizations = true
}}

# and make the special workspace to have opacity
windowrule {
  name = kitty general terminal window
  match:class = tm
  workspace = special:tm
  render_unfocused = true
  opacity = 0.95 0.95
  no_screen_share = on
}
```
